CMakeLists.txt

```
@@ -17,39 +18,51 @@ endif()
 if(APPLE)
     set(Python_FIND_FRAMEWORK NEVER)
 endif()
-find_package(Python ${Python_VERSION} EXACT REQUIRED COMPONENTS Interpreter Development)
+find_package(cpython ${Python_VERSION} QUIET COMPONENTS Interpreter Development)
+if(NOT TARGET cpython::cpython)
+    find_package(Python ${Python_VERSION} EXACT REQUIRED COMPONENTS Interpreter Development)
+else()
+    add_library(Python::Python ALIAS cpython::python)
+    set(Python_SITEARCH "${CMAKE_INSTALL_PREFIX}/lib/python3.10/site-packages")
+    set(Python_EXECUTABLE ${cpython_PACKAGE_FOLDER_RELEASE}/bin/python3)
+    set(ENV{PYTHONPATH} ${Python_SITEARCH})
+endif()
 message(STATUS "Linking and building ${project_name} against Python ${Python_VERSION}")
 
 find_package(SIP REQUIRED 6.5.0)
 
-find_package(libnest2d REQUIRED)  # The library we're creating bindings for.
+find_package(Libnest2D REQUIRED)  # The library we're creating bindings for.
 
 add_library(pynest2d INTERFACE)
+set_project_standards(pynest2d)
+use_threads(pynest2d)
+set_rpath(TARGETS
+            pynest2d
+          PATHS
+            "$<$<PLATFORM_ID:Linux>:usr/bin>"
+            "$<$<PLATFORM_ID:Linux>:usr/bin/lib>"
+            "$<$<PLATFORM_ID:Darwin>:../lib>"
+            "$<$<PLATFORM_ID:Darwin>:../Resources/lib>"
+            "../../"
```

I think this arises from how the DLLs are installed in the installation folder, e.g. that `pynest2d.so` is in the `/lib/python3.10` directory or something, and the `libnest2d.so` is in the main directory.

But it would be good to document this here, perhaps. I hope I get this right:

```suggestion
            "../../"  # In distributions of Cura, the libnest2d dependency is 2 directories up from where pynest2d gets installed.
```